nixos/tests/all-tests.nix

```diff
@@ -397,7 +397,6 @@ in
   prometheus = handleTest ./prometheus.nix {};
   prometheus-exporters = handleTest ./prometheus-exporters.nix {};
   prosody = handleTest ./xmpp/prosody.nix {};
-  prosodyMysql = handleTest ./xmpp/prosody-mysql.nix {};
```

Comment:
Needs to be removed here as well.

https://github.com/NixOS/nixpkgs/blob/b47203b28f5cf1efc4e3476ca8f333db3a2c3223/pkgs/servers/xmpp/prosody/default.nix#L89

Reply:
Crap... Thanks for the heads up.

Reply:
https://github.com/NixOS/nixpkgs/pull/155126